.github/workflows/check-changelog.yml

```diff
@@ -3,6 +3,10 @@ on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
 
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
Why are we adding additional permissions to this workflow? We don't need them for checking the changelog and blocking a PR right?

Reply:
Maybe I'm misunderstanding something, but https://github.com/mongodb/intellij/blob/d7678ff7524e3c1d9ccd1be0cae1f47efad57ab3/.github/workflows/check-changelog.yml#L14 requires write permissions for PRs to add the labels, doesn't it? Also, not that this is not adding additional permissions as the default is `write` for everything - instead we're explicitly specifying only the components we actually need and defaulting to `none` for everything else.

Reply:
Ah, then you are right. Now it makes sense that with the default permissions it works. Got it!